pages/touring-series/[slug].vue

```diff
@@ -76,34 +129,150 @@ useHead({
     }
   ]
 })
-
-// const pageClasses = computed(() => {
-//   return ['page', 'page-detail', 'page-detail--paleblue', 'page-touring-series-detail', 'page-bottom-spacer']
-// })
 </script>
 
 <template>
   <main
     id="main"
-    :class="pageClasses"
+    class="page page-detail page-detail--paleblue page-touring-series-detail"
   >
-    <SectionWrapper>
-      <div class="one-column">
-        <h3>
-          <pre>PAGE:<br/> {{ page }}</pre>
-        </h3>
-        <hr />
-        <hr />
-        <h3>
-          <pre>SERIES:<br/> {{ series }}</pre>
-        </h3>
+    <div class="one-column">
+      <NavBreadcrumb
+        class="breadcrumb"
+        :title="page?.title"
+      />
+
+      <ResponsiveImage
+        v-if="parsedImage.length === 1"
+        :media="parsedImage[0].image[0]"
+        :aspect-ratio="43.103"
+      >
+        <template
+          v-if="parsedImage[0]?.creditText"
+          #credit
+        >
+          {{ parsedImage[0]?.creditText }}
+        </template>
+      </ResponsiveImage>
+      <div
+        v-else
+        class="lightbox-container"
+      >
+        <FlexibleMediaGalleryNewLightbox
+          v-if="parsedCarouselData && parsedCarouselData.length > 0"
+          :items="parsedCarouselData"
+          :inline="true"
+        >
+          <template #default="slotProps">
+            <BlockTag :label="parsedCarouselData[slotProps.selectionIndex]?.creditText" />
+          </template>
+        </FlexibleMediaGalleryNewLightbox>
       </div>
+    </div>
+
+    <TwoColLayoutWStickySideBar>
+      <template #primaryTop>
+        <CardMeta
+          category="Series"
+          :title="page?.title"
```

Comment:
move title into the new slot we created to support h1 tags ask Jess for an example or check component library story for card meta

Reply:
<img width="706" height="212" alt="Screenshot 2025-10-21 at 5 42 25 PM" src="https://github.com/user-attachments/assets/7b94e2b6-93ba-4e85-8757-2d90f659a5b9" />